notification-portlet-webapp/src/main/webapp/scripts/modal-notice.js

```diff
@@ -157,5 +158,7 @@ if (!upmodal_notice.init) {
       // Invoke notifications
       initNotices(settings, showEachNoticeInTurn);
     }
-  })(up.jQuery);
+
+  })();
```

Comment:
without jQuery as a param this will always use global unscoped jQuery.

Reply:
It won't b/c `up.jQuery` is passed to `var launch = function($, ...)`, which surrounds everything that uses `$`.